### PR TITLE
Add template docblocks for pagination

### PR DESCRIPTION
**Pull request by Matth--**
Improved static analysis

### CODE REVIEW

File: src/Knp/Component/Pager/Pagination/AbstractPagination.php [hunk 1]
```diff
@@ -105,11 +111,17 @@ public function getPaginatorOption(string $name): mixed
         return $this->paginatorOptions[$name] ?? null;
     }
 
+    /**
```
Comment (garak) [inline src/Knp/Component/Pager/Pagination/AbstractPagination.php:114]: How is this improving static analysis?

Comment (Matth--) [reply src/Knp/Component/Pager/Pagination/AbstractPagination.php:114]: These **inheritdoc** does not really improve static analysis but refers to the type definition on the PaginationInterface.

Comment (Matth--) [reply src/Knp/Component/Pager/Pagination/AbstractPagination.php:114]: Could be removed if that is the preferred way.

Comment (Matth--) [reply src/Knp/Component/Pager/Pagination/AbstractPagination.php]: Done!

**Review by garak — APPROVED** [review]

**Pull request merged**